internal/services/v1alpha1/schema_test.go

```diff
@@ -261,7 +261,7 @@ func TestSchemaReadDeleteAndFailWrite(t *testing.T) {
 	// Issue a delete out of band for the namespace.
 	_, err = ds.ReadWriteTx(context.Background(), func(ctx context.Context, rwt datastore.ReadWriteTransaction) error {
 		for _, nsName := range requestedObjectDefNames {
```

Comment:
Given the implementation would eventually be updated to do a single sql statement, we would be loosing that by looping over here. Let's pass `requestedObjectDefNames` instead of looping?